06_create_cluster.sh

```diff
@@ -36,4 +36,15 @@ if [[ ! -z "${ENABLE_LOCAL_REGISTRY}" ]]; then
       --type merge --patch "{\"spec\":{\"samplesRegistry\":\"${LOCAL_REGISTRY_DNS_NAME}\"}}"
 fi
 
+if [[ ! -z "${APPLY_EXTRA_WORKERS}" && -f "${OCP_DIR}/extra_host_manifests.yaml" ]]; then
```

Comment:
My expectation would be that if `APPLY_EXTRA_WORKERS` has been set then the lack of `extra_hosts_manifest` should be considered an error (at least, something to be logged).
Maybe it could be useful to have a preliminary sanity check that if `APPLY_EXTRA_WORKERS` is set then `NUM_EXTRA_WORKERS` must be greater than 0?